### ISSUE TITLE
Some differrent error:Error HRESULT E_FAIL has been returned from a call to a COM component

### ISSUE THREAD

**Issue by richard008** [ffmpeginteropx/FFmpegInteropX#181]
In other issues, I found that FFmpegInteropX supports RTSP and RTMP, so I'm going to test it with:FFmpegMSSUrl = await FFmpegInteropMSS.CreateFromUriAsync(testURL);
First I test the url("rtsp://184.72.239.149/vod/mp4:BigBuckBunny_175k.mov" in unitest).
Unfortunately, I may have failed, blocked on this await method, no error reported, but also not played stream, the program blocked.
I try this url in VLC , VLC also cannot connect to this url.

///////////////////////////
Then I test the url("rtsp://wowzae2demo.streamlock.net/vod/mp4:BigBuckBunny_175k.mov" ).
Error HRESULT E_FAIL has been returned from a call to a COM component
I try this url in VLC , VLC play this rtsp url well.

////////////////////////////////

The last I test the url("rtmp://58.200.131.2:1935/livetv/hunantv",RTMP url)
It working fine..
I try this url in VLC , VLC play this RTMPurl well.
RTMP is working fine.

/////////////////////////////////////

PS: I try to run Unitest app, and it also blocked in CreateFromUri_Default() at "TestCreateFFmpegInteropMSSFromUri.cs" line 63.

How can I play the RTSP stream.

**Comment by lukasf** [comment]
The unit tests are not really working and that URL does not exist anymore. At some point we gave up trying to get these unit tests running and stable. Unit testing UWP is just a major PITA. Either the tests do not even show up, or they show up but they do not even launch. That was just not acceptable.

If an URI does not play in VLC, then I guess it is broken. I also could not play that other URI in VLC: rtsp://wowzae2demo.streamlock.net/vod/mp4:BigBuckBunny_175k.mov

**Comment by mcosmin222** [comment]
I always wondered if we could use a third party testing framework to get this rolling.

**Comment by lukasf** [comment]
Closing this. Feel free to re-open or create a new issue if you need support.
